data/seeds/users.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs");
+const faker = require("faker");
+const bcrypt = require("bcryptjs");
+
+const createFakeUser = () => ({
+  first_name: faker.name.firstName(),
+  last_name: faker.name.lastName(),
+  username: faker.internet.userName(),
+  password: faker.internet.password()
+});
+
+exports.seed = async function(knex, Promise) {
+  const fakeUsers = [];
+
+  for (let i = 0; i < 100; i++) {
+    fakeUsers.push(createFakeUser());
+  }
```

Comment:
a very smooth implementation for mass seeding -- one more reason why libraries like faker and casual are so awesome